binary/capabilities.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package io.substrait;
+
+option java_multiple_files = true;
+option csharp_namespace = "Substrait.Protobuf";
+
+// Defines a set of Capabilities that a system (producer or consumer) supports.
```

Comment:
Since the message is already capitalized and the point (I think) of this documentation is for humans to read it, the capitalization at best adds confusion.

> a system (producer or consumer)

It's fine to just say `producer or consumer`, the parenthetical isn't necessary.